amulet/level/formats/sponge_schem/format_wrapper.py

```diff
@@ -375,7 +355,7 @@ def save_to(self, f: BinaryIO):
             compact_palette, lut = brute_sort_objects_no_hash(
                 numpy.concatenate(palette)
             )
-            blocks = numpy.transpose(lut[blocks], (1, 2, 0)).ravel()  # XYZ => YZX
+            blocks = numpy.transpose(lut[blocks], (1, 2, 0)).ravel()
```

Comment:
What happened to this comment?

Reply:
I was using a VS code-extension, I must have mistakenly removed it.

Reply:
I was using a VS code-extension, I must have mistakenly removed it. Will add it back.